src/app/utils.py

```diff
@@ -417,11 +565,12 @@ def get_yet_not_approved_licenses_issues(urlType):
 
     # Remove issues with Accepted labels
     for issue in issues:
-        for label in issue.get('labels'):
-            if "new license/exception: Accepted" in label.get('name'):
-                break
-        else:
-            newRequestIssues.append(issue)
+        if issue is not None and not isinstance(issue, unicode):
```

Comment:
`if not isinstance(issue, unicode):` would be enough there. But there is another issue: are you testing your code on Python 3? Or only Python 2? this would not likely run as is on Python 3.

Reply:
I am testing the code on Python 2

Reply:
@tjasmith Here you are checking if the issue exists or not, right? I don't think so that it would return an error if the issues list is empty as well. Although, @pombredanne is right here. Please update it and I will test your PR to see nothing is broken.